packages/office-ui-fabric-react/src/common/_a11y.scss

```diff
@@ -0,0 +1,11 @@
+// To hide content while still making it readable by screen readers (Accessibility)
+@mixin ms-u-screenReaderOnly {
```

Comment:
Thanks for pointing this out, we can directly use this mixin from fabric-core in fabric-react.